### PR TITLE
feat(utils): add utc date helpers

### PR DESCRIPTION
**Pull request by dudiq**

### CODE REVIEW

File: packages/utils/types/date.d.ts [hunk 1]
```diff
@@ -10,4 +10,18 @@ declare module '@endpass/utils/date' {
   export function addToDate(date: DayJsDate, value: number, unit: string): Date;
 
   export function getWeek(date: Date): number;
+
+  export function toEqualLocalTime(date: DayJsDate): Date;
```
Comment (kazurus) [inline packages/utils/types/date.d.ts]: Dayjs? We can pass Date, timestamp...

File: packages/utils/types/date.d.ts [hunk 2]
```diff
@@ -10,4 +10,18 @@ declare module '@endpass/utils/date' {
...
+
+  export function dayBeginInUTC(date: DayJsDate): number;
```
Comment (kazurus) [inline packages/utils/types/date.d.ts]: toDate...

File: packages/utils/src/date.js [hunk 3]
```diff
@@ -25,10 +27,52 @@ const addToDate = (date, value, unit = 's') =>
...
+};
+
+const localDateFromApi = (rawObject, fieldName) => {
+  const dateValue = get(rawObject, fieldName, null);
+  const normalizedDate = dateValue ? toEqualLocalTime(dateValue * 1000) : null;
+
+  return normalizedDate;
+};
+
+const localDateToApi = (rawObject, fieldName) => {
```
Comment (kazurus) [inline packages/utils/src/date.js]: We don't need object here

File: packages/utils/types/date.d.ts (hunk 1 above)
Comment (dudiq) [reply packages/utils/types/date.d.ts]: `type DayJsDate = Date | object | string | number;`

**Pull request merged**